frontend/src/components/common/DateRangePicker.vue

```diff
@@ -1,171 +1,214 @@
 <template>
-  <!-- https://vue3datepicker.com/ -->
-  <vue-date-picker
-    v-model="choosenDates"
-    class="ml-1 mt-2 mb-3"
-    range
-    position="left"
-    :multi-calendars="MULTI_CALENDAR_OPTIONS"
-    placeholder="Datum eingeben ..."
-    :config="GENERAL_DATE_PICKER_CONFIG"
-    :text-input="TEXT_INPUT_OPTIONS"
-    :locale="LOCAL_OPTIONS"
-    :format="format"
-    :enable-time-picker="false"
-    :disabled="props.disabled"
-    :required="props.required"
-    :clearable="false"
-    :min-date="minDateProp"
-    :max-date="maxDateProp"
-    ignore-time-validation
-    :teleport="true"
-    :week-numbers="WEEK_NUMBER_OPTIONS"
-    :six-weeks="SIX_WEEK_CALENDAR_OPTIONS"
-    cancel-text="Abbrechen"
-    select-text="Datum Auswählen"
-  >
-    <template
-      #dp-input="{
-        value,
-        onInput,
-        onEnter,
-        onTab,
-        onClear,
-        onBlur,
-        onKeypress,
-        onPaste,
-        onFocus,
-      }"
-    >
-      <v-text-field
-        :label="label"
-        :model-value="value"
-        density="compact"
-        variant="underlined"
-        clearable
-        @blur="onBlur"
-        @input="onInput"
-        @click:clear="onClear"
-        @keyup.enter="onEnter"
-        @keyup.tab="onTab"
-        @keyup="onKeypress"
-        @paste="onPaste"
-        @focus="onFocus"
-      />
-    </template>
-  </vue-date-picker>
+  <div>
+    <panel-header
+      font-size="0.875rem"
+      font-weight="bold"
+      padding="10px 0 0 0"
+      header-text="Zeitraum"
+    />
+    <v-row dense>
+      <v-col cols="8">
+        <v-row
```

Comment:
Warum machst du erst eine Aufteilung mit 8 zu vier und darin wieder ne Aufteilung mit 6 zu 6? Da kannst du gleich in die erste Reihe drei Spalten mit je 4 setzen:

Spalte 1: Datepicker start
Spalte 2: Datepicker End
Spalte 3: Hinweise

Reply:
Bei der Aufteilung habe ich mich an der `ZeitauswahlRadiogroup.vue` orientiert. In den meisten Komponenten ist die erste Spaltengruppe mit dem Wert 8 anscheinen für den linken Eingabebereich vorgesehen. Die zweite Spaltengruppe mit dem Wert 4 wird dann für die Infos vorgehalten.

In der ersten achter Spaltengruppen wurden dann meistens für eine zweispaltige Darstellung nochmals einen Unterteilung in zwei Spalten mit jeweils dem Wert 6 vorgenommen.

Das wurde jedoch nicht überall umgesetzt.

![image](https://github.com/user-attachments/assets/10613692-b91d-4a8a-9680-528f0ef9195d)

Reply:
Da die Unterteilung der Spalten nicht überall gleich durchgezogen wird. Sollte da in einem LCM geradegezogen werden.

Soll dafür ein Ticket aufgemacht werden?

Reply:
Hab es beim LCM mit aufgenommen